views/editQuiz.handlebars

```diff
@@ -0,0 +1,32 @@
+<body>
+
+    {{> header}}
+
+    <div class="content_container">
+        <input class="edit_input edit_quiz_title" placeholder="Title..."></input>
+        <textarea class="edit_input edit_quiz_description" placeholder="Description..."></textarea>
+        <p class="edit_tags_label">Tags (separated by space)</p>
+        <input class="edit_input edit_quiz_tags" placeholder="Tags..."></input>
+    </div>
+
+    <div class="content_container">
+        <h4>Questions</h4>
```

Comment:
Uses many shared styles from other pages like `<h4>`, `content_container`, and `full_width_contrast_container`